app/src/main/java/com/rosario/covid19/ui/HomeActivity.kt

```diff
@@ -86,9 +88,9 @@ class HomeActivity : AppCompatActivity(), View.OnClickListener {
                 calendar.set(Calendar.YEAR, year)
                 calendar.set(Calendar.MONTH, month)
                 calendar.set(Calendar.DAY_OF_MONTH, day)
```

Comment:
If you call multiple times to the same variable maybe you could use _let_ or _apply_ to keep the code cleaner and more readable